README.md

```diff
@@ -152,6 +152,44 @@ where date_day::date >= '2019-01-01'
 group by 1
 ```
 
+### Running locally
+
+A `docker-compose` environment starts a Spark Thrift server and a Postgres database as a Hive Metastore backend.
+
+```
+docker-compose up
+```
+
+Your profile should look like this:
+
+```
+your_profile_name:
+  target: local
+  outputs:
+    local:
+      method: thrift
+      type: spark
+      schema: analytics
+      host: 127.0.0.1
+      port: 10001
```

Comment:
```suggestion
      port: 10000
```

Reply:
Ah! thanks for spotting that, that's what I have in my profile but I must have copy pasted the port from somewhere else